### PR TITLE
Text and translation updates 

### PR DESCRIPTION
**Pull request by MhewittOx**

### CODE REVIEW

File: inscriptions/ISic000333.xml [hunk 1]
```diff
@@ -185,24 +190,20 @@
...
-                    Arruntia Laudic
-                    <supplied reason="lost">e</supplied>
-                    <lb n="2"/>sibi et Aristoni<lb n="3"/>I
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <supplied reason="lost">s</supplied>uo
-                    <lb n="0"/>
-                    <gap reason="lost" quantity="1" unit="line"/>
+                    <lb n="1"/> Arruntia <g ref="#interpunct">·</g> Laudic<supplied reason="lost">e</supplied>
+                    <lb n="2"/>sibi <g ref="#interpunct">·</g> et <g ref="#interpunct">·</g> Aristoni
+                    <lb n="3"/>I <gap reason="lost" quantity="9" unit="character" precision="low"/>vo
```
Comment (JonPrag) [inline inscriptions/ISic000333.xml:197]: the 'I' should probably go in `<orig>` to mark letters clear but of which no sense can be made.

also, ideally lose the initial space before Arruntia in line 1.

File: inscriptions/ISic000333.xml [hunk 2]
```diff
@@ -185,24 +190,20 @@
...
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <supplied reason="lost">s</supplied>uo
-                    <lb n="0"/>
-                    <gap reason="lost" quantity="1" unit="line"/>
+                    <lb n="1"/> Arruntia <g ref="#interpunct">·</g> Laudic<supplied reason="lost">e</supplied>
+                    <lb n="2"/>sibi <g ref="#interpunct">·</g> et <g ref="#interpunct">·</g> Aristoni
+                    <lb n="3"/>I <gap reason="lost" quantity="9" unit="character" precision="low"/>vo          
+                    <lb n="4"/><gap reason="lost" quantity="1" unit="line"/>
                 </ab>
                 <!--This EDR text 141163 originally came via the Europeana EAGLE project, http://www.eagle-network.eu/-->
```
Comment (JonPrag) [inline inscriptions/ISic000333.xml:200]: this comment line can now be deleted.

File: inscriptions/ISic000335.xml [hunk 3]
```diff
@@ -185,14 +190,20 @@
...
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#MH">
+                <ab>
+                <lb n="1"/>Bictoria
+                <lb n="2"/>vixit an  
+                <lb n="3" break="no"/>nis <num value="4">IIII</num>
+                <lb n="4"/>meses <num value="9">VIIII</num>
```
Comment (JonPrag) [inline inscriptions/ISic000335.xml:198]: I would suggest that one would encode the erroneous/non-standard form of 'menses' thus:
`<choice><orig>meses</orig><reg>menses</reg></choice>`

Comment (JonPrag) [reply inscriptions/ISic000335.xml:198]: Further, you could do the same thing to Bictoria/Victoria in line 1?

File: inscriptions/ISic000338.xml [hunk 4]
```diff
@@ -185,41 +190,22 @@
...
-                        <ex>arco</ex>
-                    </expan>
-                    Caepario<lb n="3"/>Sotericho et<lb n="4"/>filiae eius
-                    <lb n="5"/>
-                    <expan>
-                        <abbr>M</abbr>
-                        <ex>arcus</ex>
-                    </expan>
-                    Capillianus<lb n="6"/>fecit
+                    <lb n="1"/><expan><abbr>D</abbr><ex>is</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>M</abbr><ex>anibus</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>M</abbr><ex>anibus</ex></expan>
```
Comment (JonPrag) [inline inscriptions/ISic000338.xml:195]: I think you've ended up repeating Manibus by mistake? DMS?

File: inscriptions/ISic000338.xml [hunk 5]
```diff
@@ -185,41 +190,22 @@
...
+                    <lb n="2"/><expan><abbr>M</abbr><ex>arcus</ex></expan> <g ref="#interpunct">·</g> Caepario
```
Comment (JonPrag) [inline inscriptions/ISic000338.xml:196]: Marco?

File: inscriptions/ISic000338.xml [hunk 6]
```diff
@@ -185,41 +190,22 @@
...
+                    <lb n="3"/>Sotericho <g ref="#interpunct">·</g> et
+                    <lb n="4"/>filiae <g ref="#interpunct">·</g> eius
+                    <lb n="5"/><expan><abbr>M</abbr><ex>arcus</ex></expan> Capillianus
+                    <lb n="6"/>fecit
                 </ab>
                 <!--This EDR text 141165 originally came via the Europeana EAGLE project, http://www.eagle-network.eu/-->
```
Comment (JonPrag) [inline inscriptions/ISic000338.xml:202]: you can delete this comment line

File: inscriptions/ISic000339.xml [hunk 7]
```diff
@@ -186,46 +191,21 @@
...
-                    </expan>
-                    <expan>
-                        <abbr>M</abbr>
-                        <ex>anibus</ex>
-                    </expan>
-                    <expan>
-                        <abbr>s</abbr>
-                        <ex>acrum</ex>
-                    </expan>
+                    <lb n="1"/><expan><abbr>D</abbr><ex>is</ex></expan> <g ref="#interpunct">·</g><expan><abbr>M</abbr><ex>anibus</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>s</abbr><ex>acrum</ex></expan>
```
Comment (JonPrag) [inline inscriptions/ISic000339.xml:196]: add a space after the closing `</g>` of the first interpunct

File: inscriptions/ISic000339.xml [hunk 8]
```diff
@@ -186,46 +191,21 @@
...
-                    VIII
-                    <lb n="4"/>
-                    <expan>
-                        <abbr>fec</abbr>
-                        <ex>it</ex>
-                    </expan>
-                    pater<lb n="5"/>Lupercus.
+                    <lb n="3"/><expan><abbr>vix</abbr><ex>it</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>m</abbr><ex>ensibus</ex></expan> <g ref="#interpunct">·</g> <num value="8">VIII</num>
+                    <lb n="4"/><expan><abbr>fec</abbr><ex>it</ex></expan> <g ref="#interpunct">·</g> pater
+                    r<lb n="5"/>Lupercus.
```
Comment (JonPrag) [inline inscriptions/ISic000339.xml:200]: rogue r before the `<lb>`? And delete the final period.

File: inscriptions/ISic000339.xml [hunk 9]
```diff
@@ -186,46 +191,21 @@
...
                 </ab>
                 <!--This EDR text 141166 originally came via the Europeana EAGLE project, http://www.eagle-network.eu/-->
```
Comment (JonPrag) [inline inscriptions/ISic000339.xml:202]: can deletet eh comment line

File: inscriptions/ISic000343.xml [hunk 10]
```diff
@@ -185,45 +190,21 @@
...
-                        <ex>os</ex>
-                    </expan>
-                    XII<lb n="5"/>pater fecit.
+                    <lb n="1"/><expan><abbr>D</abbr><ex>is</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>M</abbr><ex>anibus</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>S</abbr><ex>acrum</ex></expan>
+                    <lb n="2"/><expan><abbr>T</abbr><ex>itus</ex></expan> <g ref="#interpunct">·</g> Clatius He
+                    <lb n="3" break="no"/>resistratus
+                    <lb n="4"/><expan><abbr>vix</abbr><ex>it</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>ann</abbr><ex>os</ex></expan> <g ref="#interpunct">·</g> <num value="12">XII</num>
+                    <lb n="5"/>pater <g ref="#interpunct">·</g> fecit
                 </ab>
                 <!--This EDR text 141170 originally came via the Europeana EAGLE project, http://www.eagle-network.eu/-->
```
Comment (JonPrag) [inline inscriptions/ISic000343.xml:201]: can delete comment line

**Review by JonPrag — CHANGES_REQUESTED** [review]
Just some very small changes / suggestions - generally excellent!

**Comment by MhewittOx** [comment]
Thanks, hopefully that should all be resolved now!

**Review by JonPrag — APPROVED** [review]
all good, thanks!

**Pull request merged**